internal/receiver/databricksreceiver/metadata.yaml

```diff
@@ -62,3 +79,1082 @@ metrics:
+      [cluster_id, app_id, executor_id]
+  databricks.spark.job.num_tasks:
+    enabled: true
+    description: n/a
+    unit: "{tasks}"
+    gauge:
+      value_type: int
+    attributes:
+      [cluster_id, app_id, spark_job_id]
+  databricks.spark.job.num_active_tasks:
+    enabled: true
+    description: n/a
+    unit: "{tasks}"
+    gauge:
+      value_type: int
+    attributes:
+      [cluster_id, app_id, spark_job_id]
+  databricks.spark.job.num_completed_tasks:
+    enabled: true
+    description: n/a
+    unit: "{tasks}"
+    gauge:
+      value_type: int
+    attributes:
+      [cluster_id, app_id, spark_job_id]
+  databricks.spark.job.num_skipped_tasks:
+    enabled: true
+    description: n/a
+    unit: "{tasks}"
+    gauge:
+      value_type: int
+    attributes:
+      [cluster_id, app_id, spark_job_id]
+  databricks.spark.job.num_failed_tasks:
+    enabled: true
+    description: n/a
+    unit: "{tasks}"
+    gauge:
+      value_type: int
+    attributes:
+      [cluster_id, app_id, spark_job_id]
+  databricks.spark.job.num_active_stages:
+    enabled: true
+    description: n/a
+    unit: "{stages}"
+    gauge:
+      value_type: int
+    attributes:
+      [cluster_id, app_id, spark_job_id]
+  databricks.spark.job.num_completed_stages:
+    enabled: true
+    description: n/a
+    unit: "{stages}"
+    gauge:
+      value_type: int
+    attributes:
+      [cluster_id, app_id, spark_job_id]
+  databricks.spark.job.num_skipped_stages:
+    enabled: true
+    description: n/a
+    unit: "{stages}"
+    gauge:
+      value_type: int
+    attributes:
+      [cluster_id, app_id, spark_job_id]
+  databricks.spark.job.num_failed_stages:
+    enabled: true
+    description: n/a
+    unit: "{stages}"
+    gauge:
+      value_type: int
+    attributes:
+      [cluster_id, app_id, spark_job_id]
+  databricks.spark.stage.executor_run_time:
+    enabled: true
+    description: n/a
+    unit: ""
+    gauge:
+      value_type: int
+    attributes:
+      [cluster_id, app_id, spark_job_id]
+  databricks.spark.stage.input_bytes:
+    enabled: true
+    description: n/a
+    unit: ""
+    gauge:
+      value_type: int
+    attributes:
+      [cluster_id, app_id, spark_job_id]
+  databricks.spark.stage.input_records:
+    enabled: true
+    description: n/a
+    unit: ""
+    gauge:
+      value_type: int
+    attributes:
+      [cluster_id, app_id, spark_job_id]
+  databricks.spark.stage.output_bytes:
+    enabled: true
+    description: n/a
+    unit: "By"
+    gauge:
+      value_type: int
+    attributes:
+      [cluster_id, app_id, spark_job_id]
+  databricks.spark.stage.output_records:
+    enabled: true
+    description: n/a
+    unit: ""
+    gauge:
+      value_type: int
+    attributes:
+      [cluster_id, app_id, spark_job_id]
+  databricks.spark.stage.memory_bytes_spilled:
+    enabled: true
+    description: n/a
+    unit: "By"
+    gauge:
+      value_type: int
+    attributes:
+      [cluster_id, app_id, spark_job_id]
```

Comment:
`cluster_id` and `app_id` seem to be present for all metrics, worth moving to resource attributes.
I see all metrics are enabled by default, is that expected, is there more to do there?